RecoParticleFlow/PFTracking/python/particleFlowTrackWithConversion_cff.py

```diff
@@ -22,9 +22,10 @@
 #HON#pfConversions.OtherOutInCollection      =           cms.VInputTag(cms.InputTag("generalTracks"))
 #HON#pfConversions.OtherInOutCollection      =           cms.VInputTag(cms.InputTag("generalTracks"))
 
-particleFlowTrackWithConversion =cms.Sequence(
-    pfTrackElec*
-    #HON#trackerOnlyConversionSequence*
```

Comment:
Sorry @jeongeun , but at a closer look I realise that these sequences in the commented out lines are very old, already removed from CMSSW_4_2_0 since January 2011
At this point, it is probably worth profiting of this PR and clean up this config by removing all commented out lines.
After that, the PR is ready to go (tests already succeded, and that won't change after removing comments).